### PR TITLE
Completed JavaScript Conditionals exercise.

### PR DESCRIPTION
**Pull request by dennis-pnguyen**

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+function isOldEnoughToDrinkAndDrive(person){
+    if (person) {
+        return false;
+    }
+};
+
+function categorizeAcidity(pH){
+    if (pH === 7){
+        return 'neutral';
+    } else if (pH === 0 && pH <= 7) {
```
Comment (shawnkost) [inline javascript-conditionals/main.js]: This should be `>= 0` & `< 7`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Nicely done Dennis! Just one tweak here

File: javascript-conditionals/main.js (hunk 1 above)
Comment (dennis-pnguyen) [reply javascript-conditionals/main.js]: javascript-conditionals

Updated, thanks!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**